config/astral.yml

```diff
@@ -1,23 +1,34 @@
+# Astral configuration
+# Note that values can be supplied here or as environment vars (UPPER_CASE).
 shared:
-  vault_addr: 
   vault_token:
+  vault_addr:
+  # if VAULT_ADDR is https with self-signed cert, need to provide
+  # CA cert (path to file)
```

Comment:
This comment confused me for a bit.  It would have been clearer to me if it read:

```
 # if VAULT_ADDR is https with self-signed cert, need to provide
  # CA cert (path to file) in "vault_ssl_cert" below:
```

Reply:
updated if you can please re-up the approval